.openshiftio/application.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
     description: This template creates a Build Configuration using an S2I builder.
     tags: instant-app
 parameters:
+  - name: SUFFIX_NAME
+    description: The suffix name for the template objects
+    displayName: Suffix name
+    value: master
+  - name: RELEASE_VERSION
+    description: The release version number of application
+    displayName: Release version
+    value: 1
```

Comment:
The version label values that this replaces, are sort of tied to the version in the package.json.  So the master branch has 2..x.x and the 8.x branch has 1.x.x .  If possible, i would like to keep the values the same.

Reply:
Sure we can.